tox.ini

```
@@ -0,0 +1,28 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py{35}-django{22}, py{38}-django{22,30}, py38-quality
+
+[testenv]
+
+setenv =
+    PYTHONPATH = {toxinidir}/tests/utils
+
+deps =
+    django22: Django==2.2.12
```

nit: [use juniper's django version](https://github.com/edx/edx-platform/blob/open-release/juniper.1/requirements/edx/django.txt):
```suggestion
    django22: Django==2.2.13
```

---

Yeah, sorry about that. I should've paid attention to that.